### ISSUE TITLE
Conflict with NetworkAnalyzer

### ISSUE THREAD

**Issue by agitter** [Murali-group/PathLinker-Cytoscape#13]
If the Cytoscape NetworkAnalyzer is run before PathLinker, PathLinker will hang with the message _PathLinker is running_.  NetworkAnalyzer creates an edge attribute `EdgeBetweenness`, and PathLinker does not allow this attribute to exist prior to its execution.  With the attribute present, I get the error message:
```
Cytoscape 3.2.0> Exception in thread "AWT-EventQueue-0" java.lang.IllegalArgumentException: column already exists with name: 'EdgeBetweenness' with type: class java.lang.Double
in table: Sheet1 default edge
        at org.cytoscape.model.internal.SharedTableFacade.checkIfAlreadyExists(SharedTableFacade.java:92)
        at org.cytoscape.model.internal.SharedTableFacade.createColumn(SharedTableFacade.java:112)
        at org.cytoscape.model.internal.LocalTableFacade.createColumn(LocalTableFacade.java:90)
        at org.cytoscape.model.internal.LocalTableFacade.createColumn(LocalTableFacade.java:83)
        at com.dpgil.pathlinker.path_linker.internal.PathLinkerPanel.createKSPSubgraph(PathLinkerPanel.java:931)
        at com.dpgil.pathlinker.path_linker.internal.PathLinkerPanel.runKSP(PathLinkerPanel.java:322)
        at com.dpgil.pathlinker.path_linker.internal.PathLinkerPanel.access$1(PathLinkerPanel.java:265)
        at com.dpgil.pathlinker.path_linker.internal.PathLinkerPanel$1.run(PathLinkerPanel.java:237)
        at java.awt.event.InvocationEvent.dispatch(Unknown Source)
        at java.awt.EventQueue.dispatchEventImpl(Unknown Source)
        at java.awt.EventQueue.access$500(Unknown Source)
        at java.awt.EventQueue$3.run(Unknown Source)
        at java.awt.EventQueue$3.run(Unknown Source)
        at java.security.AccessController.doPrivileged(Native Method)
        at java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(Unknown Source)
        at java.awt.EventQueue.dispatchEvent(Unknown Source)
        at java.awt.EventDispatchThread.pumpOneEventForFilters(Unknown Source)
        at java.awt.EventDispatchThread.pumpEventsForFilter(Unknown Source)
        at java.awt.EventDispatchThread.pumpEventsForHierarchy(Unknown Source)
        at java.awt.EventDispatchThread.pumpEvents(Unknown Source)
        at java.awt.EventDispatchThread.pumpEvents(Unknown Source)
        at java.awt.EventDispatchThread.run(Unknown Source)
```

If I then delete the `EdgeBetweenness` attribute, PathLinker runs successfully.  I encountered the problem with Cytoscape 3.2.0 and PathLinker 1.2.4.

**Comment by jlaw9** [comment]
I was able to reproduce the problem on Cytoscape 3.4.0.

PathLinker copies all of the edge columns and attributes to the new subnetwork, but before it creates the columns in the new network, it checks to see if it already exists. It seems like the problem isn't on our end. I'm contacting the NetworkAnalyzer tool builders to see if it's something on their end that they can fix.

**Comment by jlaw9** [comment]
This issue was fixed by fixing issue #12. Basically, instead of creating an empty subnetwork and then copying the attributes to it, the code just creates a subnetwork of the original with a given list of nodes and edges, which is much simpler anyway.

**Comment by agitter** [comment]
@jlaw9 Thanks for working on all of these issues!
